### ISSUE TITLE
Making levelling up less prone to power-leveling exploits

### ISSUE THREAD

**Issue by eliotcougar** [StrangeLoopGames/EcoSuggestions#730]
Before 8.0 we had an issue when people didn't use their professions until they've got max efficiency.
Now we have another balance issue: people find ways to level-up professions by making the most cheapest stuff, and performing actions that are easiest to mass-perform in order to level-up to Level 7... Only after that they start actually using the profession...
The examples are: cutting grass to quickly reach Gathering 7, crafting the cheapest dish, made from the most common, easily replenishable ingredient available to level-up campfire cooking and other types of cooking, render fat to get Level 7 cooking quickly... Mass-producing pitch levels up mortaring very easily... Grinding is the most boring one to level up - just make 80-100 units of flour... You get the idea...

The proposal is to limit the maximum amount of total XP you can get from every single crafting recipe... So, for example, if you only craft hundreds of units of pitch, you can't get to level 7 masonry, but only to level 2, maybe. The amount of XP you can get from a single recipe can be adjusted so that in order to reach Level 7, you'd need to craft a decent amount of at least half of the available recipes from that skill...
Level 7 means mastering the profession - you shouldn't be able to master it by performing only one action associated with the profession.

Variation for Gathering would imply gathering at least half of the available crops, not just grass and huckleberries... And for hunting it may be even reasonable to require players to hunt all the main animals at least once or twice, or the total amount ox XP you can get from each animal can be proportional to the amount of meat you get from it...

Bottom line is: there should be no exploits that allow power-leveling professions by doing only one task dozens of times...

**Comment by feildmaster** [comment]
I'm not entirely sure I agree with your statement of "you shouldn't be able to master a profession by doing a single action", you certainly are able to master the basics in order to further your understanding of a profession. In some professions mastering the basics literally is mastery of the task so long as you also apply your mind to further reflect on those actions.

While it makes sense to perhaps decrease the experience gained from certain recipes as you gain levels, by no means should an action *never* give you progress.

**Comment by eliotcougar** [comment]
The problem is mainly that it's too easy to just queue massive amount of one item, go away for half an hour, and return with Level 7 skill, and then start using that skill to make other more valuable items more efficiently... The idea behind the 8.0 changes was to prevent players from not using the skill until they've got max efficiency in it... But the way it is now is not much different from it... The only difference is that you have to find a single recipe that gives you the most bang for your buck, the one that you don't care if you waste materials due to inefficiency...

You get to level 7 so quickly by exploiting these technique (less than an hour usually) that for the rest of the game most of the stuff you make never gives you progress, because you've got all of it by cutting grass or rendering fat for 30 minutes...

**Comment by Deantwo** [comment]
I would love to see resource processing skills and end product crafting skills split up. Such as separating Hewing and Woodworking.
So one makes the hewn logs and the other makes the furniture and other things using those hewn logs.

This is even more desired for Smelting and Metalworking. Right now the smith can do almost everything with just a single skill. Make smelting ingots and working with said ingots into separate professions.

**Comment by eliotcougar** [comment]
> Hewing and Woodworking.
> Smelting and Metalworking.

And if the idea of having some amount of scrap material produced in every woodworking and metalworking recipe #731 , depending on the skill level, is implemented, that would add some interplay between them, and will make it more desirable to give materials to more proficient craftsmen...
Hewing produces hewn logs, boards, decorative wood
Woodworker makes furniture and tools from those, and scrap wood and sawdust as a waste product
Hewing takes scrap wood and turns it into wood pulp or fuel, or something else
Smelter makes ingots, metalworker makes items and scrap metal as a waste product, smelter takes scrap metal and smelts it back into ingots... So, higher skills and making less scrap metal means less energy wasted on recycling scrap metal...

**Comment by Blanclour** [comment]
Having certain professions split up such as the ones mentioned above would benefit server populations. New players would have more niches to fill rather then being forced to compete with already established players who already have high efficiency.
